Software_Code/Admin/template.php

```diff
@@ -0,0 +1,273 @@
+<!doctype html>
+<html lang="en">

```

Comment:
What is this and what purpose does it serve? It seems to be using old map code too?